apps/web/pages/booking/[uid].tsx

```diff
@@ -146,7 +146,7 @@ export default function Success(props: SuccessProps) {
   const shouldAlignCentrallyInEmbed = useEmbedNonStylesConfig("align") !== "left";
   const shouldAlignCentrally = !isEmbed || shouldAlignCentrallyInEmbed;
   const [calculatedDuration, setCalculatedDuration] = useState<number | undefined>(undefined);
-
+  const { requiresLoginToUpdate } = props;
```

Comment:
Created this new flag to know if login is required to cancel/reschedule booking. This will only happen if booking is seated event.